astropy/logger.py

```diff
@@ -28,11 +29,12 @@
 else:
     _WITHIN_IPYTHON = True
 
-__all__ = ["Conf", "conf", "log", "AstropyLogger", "LoggingError"]
-
-# import the logging levels from logging so that one can do:
-# log.setLevel(log.DEBUG), for example
-logging_levels = [
+__all__ = [
+    "Conf",
+    "conf",
+    "log",
+    "AstropyLogger",
+    "LoggingError",
```

Comment:
Not sure if this syntax is allowed but would be nice to keep the comment.

```suggestion
    "LoggingError",
    # import the logging levels from logging so that one can do:
    # log.setLevel(log.DEBUG), for example
```